main.go

```diff
@@ -284,7 +288,9 @@ func main() {
 		}
 
 		//@TODO This is a hack to keep compatibility now, to be deleted.
-		go http.ListenAndServe(fmt.Sprintf(":%v", Config.HttpPort), h)
+		go func() {
```

Comment:
just for me to understand/learn. what is the reason to switch from single line to function block?

Reply:
err was not checked, and lint complains about. 

I added this mainly because it's going to be deleted in the following days.

Reply:
intersting. thanks.